sensehat/README.md

```diff
@@ -4,7 +4,7 @@ Sense Hat driver for Android Things
 This driver provides easy access to the peripherals available on the Raspberry Pi [Sense Hat][product]:
 - 8x8 LED matrix
 - TODO: 5 buttons joystick
-- TODO: Sensors
+- TODO: Sensors other then LPS25H
```

Comment:
I'd rephrase this list:

- 8x8 LED matrix
- LPS25H
...
- TODO: Other sensors